web/client/components/share/api-template.raw

```diff
@@ -26,6 +27,19 @@
             height: 100%;
         }
         </style>
+        <script type="text/javascript">
+            var browserInfo = bowser.getParser(window.navigator.userAgent);
+            var isValidBrowser = browserInfo.satisfies({
+                "edge": ">1",
+                "chrome": ">1",
+                "safari": ">1",
+                "firefox": ">1"
+            });
+            if (!isValidBrowser) {
+                window.location.href = "unsupportedBrowser.html"
```

Comment:
This will not work. Typically API is used on a separate page, so in this case unsupportedBrowser.html is not reachable.